chemprop/cli/train.py

```diff
@@ -611,39 +598,15 @@ def main(args):
         spectral_activation=args.spectral_activation,
     )
 
-    if isinstance(predictor_ffn, RegressionFFN):
-        scaler = train_dset.normalize_targets()
-        val_dset.normalize_targets(scaler)
-        logger.info(f"Train data: loc = {scaler.mean_}, scale = {scaler.scale_}")
-    else:
-        scaler = None
-
-    train_loader = data.MolGraphDataLoader(train_dset, args.batch_size, args.num_workers)
-    val_loader = data.MolGraphDataLoader(val_dset, args.batch_size, args.num_workers, shuffle=False)
-
-    if multicomponent:
-        if len(test_data[0]) > 0:
-            test_dsets = [make_dataset(data, args.rxn_mode) for data in test_data]
-            test_dset = data.MulticomponentDataset(test_dsets)
-            test_loader = data.MolGraphDataLoader(
-                test_dset, args.batch_size, args.num_workers, shuffle=False
-            )
-        else:
-            test_loader = None
-    else:
-        if len(test_data) > 0:
-            test_dset = make_dataset(test_data, args.rxn_mode)
-            test_loader = data.MolGraphDataLoader(
-                test_dset, args.batch_size, args.num_workers, shuffle=False
-            )
-        else:
-            test_loader = None
+    if args.loss_function is None:
+        logger.info(
+            f"No loss function was specified! Using class default: {predictor_cls.criterion}"
```

Comment:
```suggestion
            f"No loss function was specified! Using class default: {predictor_cls._default_criterion}"
```

Reply:
I originally suggested this change, but had to push it myself to get the tests to run. If you wanted to fix it a different way, that is fine too.